migrations/4_deploy_safe_old.js

```diff
@@ -0,0 +1,14 @@
+const truffleContract = require('@truffle/contract');
```

Comment:
yeah, it was installed when I thought we could have support both contract versions... shall we create such a test?

Reply:
I think that test should be elsewhere, like in the core repo

Reply:
Ok, if we are keeping the old contracts I actually prefer that we deplloy the new contracts in the 4th migration script. So that we preserve the cronological order which represents the real stages.

Reply:
ah ok, then I will change this